doc/jobs/devel_jobs.rst

```diff
@@ -183,68 +183,12 @@ Instead of invoking the generated script it can also be *sourced*:
 
 .. code:: sh
 
-  . devel_job_indigo_roscpp_core.sh
+  . devel_job_rolling_rclcpp.sh
 
 The return code of the invocation of ``catkin_tests_results`` /
 ``colcon test-result``is then available in the environment variable
 ``test_result_RC``.
 
-Run the *devel* job on Travis
```

Comment:
This section could be adapted to GitHub Actions and there are examples in this very repo.

Reply:
Ditto.